### PR TITLE
Updated Command descriptions 

### LINKED ISSUE #7603: [mesheryctl] Update naming convention
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

There are many instances where features are named as `Service Mesh` related.

For example:
```
mesheryctl --help
Meshery is the service mesh management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.

Usage:
  mesheryctl [flags]
  mesheryctl [command]

Examples:

// Base command
mesheryctl

// Display help about command/subcommand
mesheryctl --help
mesheryctl system start --help

// For viewing verbose output
mesheryctl -v [or] --verbose
	

Available Commands:
  app         Service Mesh Apps Management
  completion  Output shell completion code
  exp         Experimental commands for mesheryctl
  help        Help about any command
  mesh        Service Mesh Lifecycle Management
  pattern     Service Mesh Patterns Management
  perf        Performance Management
  system      Meshery Lifecycle Management
  version     Version of mesheryctl

Flags:
      --config string   path to config file (default "/Users/azanulhaque/.meshery/config.yaml")
  -h, --help            help for mesheryctl
  -v, --verbose         verbose output

Use "mesheryctl [command] --help" for more information about a command.
```

#### Desired Behavior

All the `Service Mesh` instances need to be `Cloud Native` instead.

#### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by thisiskaransgit**
**Notes for Reviewers**

This PR fixes #7603 

Replaced `Service Mesh` with `Cloud Native`, in all instances where the changes were required.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/internal/cli/root/root.go [hunk 1]
```diff
@@ -46,7 +46,7 @@ var (
 var RootCmd = &cobra.Command{
 	Use:   "mesheryctl",
 	Short: "Meshery Command Line tool",
-	Long:  `Meshery is the service mesh management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.`,
+	Long:  `Meshery is the Cloud Native management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.`,
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/root.go:49]: lowercase, please. Also, update the second "service mesh" reference in the sentence.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Thanks for this, @thisiskaransgit

File: mesheryctl/internal/cli/root/root.go (hunk 1 above)
Comment (thisiskaransgit) [reply mesheryctl/internal/cli/root/root.go]: lowercased and updated the reference.

**Review by alphaX86 — APPROVED** [review]
LGTM 👍

File: mesheryctl/internal/cli/root/root.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ var (
 var RootCmd = &cobra.Command{
 	Use:   "mesheryctl",
 	Short: "Meshery Command Line tool",
-	Long:  `Meshery is the service mesh management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.`,
+	Long:  `Meshery is the cloud native management plane, providing lifecycle, performance, and configuration management of Kubernetes, any service mesh, and their workloads.`,
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/root.go:49]:
```suggestion
	Long:  `As a self-service engineering platform, Meshery enables collaborative design and operation of cloud native infrastructure.`,
```

**Review by leecalcote — APPROVED** [review]
Good deal, @thisiskaransgit 

For future reference, note that I updated one item, taking the text from the docs.meshery.io landing page.

**Pull request merged**